mon-pix/app/components/challenge-item-qroc.hbs

```diff
@@ -60,9 +60,7 @@
                   data-uid="qroc-proposal-uid"
                   data-test="challenge-response-proposal-selector"
                 >
-                  <:label>
-                    <MarkdownToHtml @isInline={{true}} @extensions="remove-paragraph-tags" @markdown={{block.text}} />
```

Comment:
du coup, je me demande a quoi à pu servir ce mardowntoHtml à l'époque. on avait des fois des label vide en entete de page ? 🤔 .

Je pense qu'il faudrait plus faire 

if block.text { markdownTohtml } else { this.buildGenericLabel index block.ariaLabel }

Après c'est en screenReaderOnly du coup ... quel utilité d'avoir du html 🤔 .


il faudrait trouver un QROC avec du text que l'on souhaite afficher si ça existe.